frontend/src/App.js

```diff
@@ -411,6 +413,30 @@ export default function App() {
                   component={() => <RoutineReports />}
                   role="Reports"
                 />
+                <SecureRoute
+                  path="/Report"
+                  exact
+                  component={() => <RoutineIndex />}
+                  role="Reports"
+                />
+                <SecureRoute
+                  path="/StatisticsReport"
+                  exact
+                  component={() => <RoutineIndex />}
+                  role="Reports"
+                />
+                <SecureRoute
+                  path="/SummaryOfAllTests"
+                  exact
+                  component={() => <RoutineIndex />}
+                  role="Reports"
+                />
+                <SecureRoute
+                  path="/HIVTestSummary"
+                  exact
+                  component={() => <RoutineIndex />}
```

Comment:
Since all these paths are redirecting to the same component ie RoutineIndex , why dont you use a single path ie `/AgregateReports` to load the same component and load the diferent reports based on the report param

Reply:
Sounds good!